internal/core/common/ecs/ecs.go

```diff
@@ -0,0 +1,45 @@
+package ecs
+
+import "strings"
+
+// TaskMetadata contains a set of properties from ECS Task Metadata
+type TaskMetadata struct {
+	ClusterName string      `json:"Cluster"`
+	TaskARN     string      `json:"TaskARN"`
+	Family      string      `json:"Family"`
+	Revision    string      `json:"Revision"`
+	KnownStatus string      `json:"KnownStatus"`
+	Containers  []Container `json:"Containers"`
+}
+
+// GetDimensions returns a set of dimensions based on the task metadata
+func (task *TaskMetadata) GetDimensions() map[string]string {
+	dims := make(map[string]string, 0)
+	if idx := strings.Index(task.ClusterName, "/"); idx >= 0 {
+		dims["ClusterName"] = task.ClusterName[idx+1 : len(task.ClusterName)]
```

Comment:
If `task.ClusterName` is `my-cluster/` what will happen?  I think it will panic since `idx+1` will be out of bounds to the slice.

Reply:
Also, why is the stuff after the first slash undesirable?

Reply:
Every time I saw the cluster name provided by metadata API, task.ClusterName was having a format of `[AWS task ARN]/[Cluster name given by user]` and that was why I was taking whatever comes after the first slash. If the same happens all the time, when user set `my-cluster/` as the cluster name, it will still be parsed as `my-cluster/` after the task ARN part is removed.

Reply:
I just did another round of research and there wasn't really any documentation that guarantee the task ARN will be a part of the cluster name, so now we should consider when the cluster name is solely what's given by user. I have three options to resolve this.

Reply:
1. Just use the whole thing ex) `arn:aws:ecs:us-east-1:906383545488:cluster/seon-fargate-test`
2. Use regex to extract only the resource name  ex) `seon-fargate-test`
3. TaskARN will be given thru another label so replace TaskARN in the cluster name with an empty string

Reply:
ok, yeah it's fine that the Task ARN isn't part of the cluster name because we already get that separately.

Keep this logic you have for `ClusterName`, but guard it to avoid panics in case of a malformed ClusterName field.

Then send in the `task.ClusterName` field unaltered as `ecs_cluster_arn`.